src/pybind/mgr/orchestrator/module.py

```diff
@@ -964,6 +964,18 @@ def _set_prometheus_access_info(self, username: Optional[str] = None, password:
         except ArgumentError as e:
             return HandleCommandResult(-errno.EINVAL, "", (str(e)))
 
+    @_cli_write_command('orch prometheus set-target')
+    def _set_prometheus_target(self, url: Optional[str] = None, inbuf: Optional[str] = None) -> HandleCommandResult:
+        completion = self.set_prometheus_target(url)
+        result = raise_if_exception(completion)
+        return HandleCommandResult(stdout=json.dumps(result))
+
+    @_cli_write_command('orch prometheus remove-target')
+    def _remove_prometheus_target(self, url: Optional[str] = None, inbuf: Optional[str] = None) -> HandleCommandResult:
```

Comment:
similar comment to to the one about set-target regarding the optional url arg and unused inbuf arg.